src/update.ts

```diff
@@ -0,0 +1,163 @@
+import { Update as U, Message as M } from './core/types/typegram'
+
+/**
+ * Update types to use with `Context<Update>`.
+ * The exact structure of this export may change in the future, or be removed entirely.
+ * @experimental
+ */
+// eslint-disable-next-line @typescript-eslint/no-namespace
+export namespace Update {
+  export interface Message extends U.MessageUpdate {}
+  export interface EditedMessage extends U.EditedMessageUpdate {}
+  export interface ChannelPost extends U.ChannelPostUpdate {}
+  export interface EditedChannelPost extends U.EditedChannelPostUpdate {}
+  export interface InlineQuery extends U.InlineQueryUpdate {}
+  export interface ChosenInlineResult extends U.ChosenInlineResultUpdate {}
+  export interface CallbackQuery extends U.CallbackQueryUpdate {}
+  export interface ShippingQuery extends U.ShippingQueryUpdate {}
+  export interface PreCheckoutQuery extends U.PreCheckoutQueryUpdate {}
+  export interface Poll extends U.PollUpdate {}
+  export interface PollAnswer extends U.PollAnswerUpdate {}
+  export interface MyChatMember extends U.MyChatMemberUpdate {}
+  export interface ChatMember extends U.ChatMemberUpdate {}
+  export interface ChatJoinRequest extends U.ChatJoinRequestUpdate {}
+
+  export type All =
```

Comment:
What's the difference between `All` and `U`?

Reply:
Vanity - type names will appear as defined here rather than source (Message instead of MessageUpdate).

Reply:
Why not `type All = U` then?

Reply:
Gotcha